.github/workflows/release.yml

```diff
@@ -9,11 +9,15 @@ on:
 # Always wait for previous release to finish before releasing again
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
+env:
+  ORG: bobanetwork
+  NAME: v3-anchorage
+
 jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
-    if: github.repository == 'ethereum-optimism/optimism'
+    if: github.repository == 'bobanetwork/v3-anchorage'
```

Comment:
Should this be `{{env.NAME}}`?

Reply:
```
    Because when statements are evaluated when the configuration is compiled prior to running, and environment variables are determined at runtime, the method I suggested won’t work as is.

However, you can get around this limitation by using [pipeline parameters 207](https://circleci.com/docs/2.0/pipeline-variables/#pipeline-parameters-in-configuration) with [dynamic config 261](https://circleci.com/docs/2.0/dynamic-config/). I would also recommend the “[Intro to Dynamic Config via Setup Workflows 283](https://discuss.circleci.com/t/intro-to-dynamic-config-via-setup-workflows/39868)” Support Solution posted by one of my colleagues.
```

will take a look at the second option though!